README.md

```diff
@@ -4,3 +4,105 @@ I am a simple downloader that downloads images from URLs in a CSV and names them
 
 ## Requirements
 Python 3.7+
+
+## Installation
+```bash
+pip install git+https://github.com/Imageomics/cautious-robot
+```
+
+## How it Works
+
+Cautious-robot will check the provided CSV for `IMG_NAME`, `URL`, and `SUBFOLDERS` (if provided), then download all images that have a value in the `IMG_NAME` column. Note that choice of image filename should be unique , especially if working in a flat directory, as the images will otherwise be overwritten. Images that have a filename but no `URL` are recorded in the error log. If desired, a secondary output directory will be created with square copies of the images downsampled to the desired size (e.g., 256 x 256). Parameters such as time to wait between retries, the maximum number of times to try retrying an image, and which index of the CSV to start with can all also be passed. Cautious-robot will retry image downloads when the following responses are returned: `429, 500, 502, 503, 504`.
```

Comment:
Uniqueness of `filename` (or `subdir/filename`) should be confirmed prior to the download start.